docs/CNVs/CNV-methods.tex

```diff
@@ -670,125 +712,170 @@ \subsection{Generative Model for Read Counts} \label{coverage-model}
+\begin{equation}
+\mathcal{L}(\vPsi) = \mathcal{L}(\vPsi_0) + \alpha_t \, (\Psi_t - \Psi_{0,t}) + \frac{1}{2} \, \beta_t \, (\Psi_t - \Psi_{t,0})^2 + \ldots
+\end{equation}
+where:
 \begin{align}
-\sum_s \left[ \frac{1}{\Psi_t + \Sigma_{st}} - \frac{\beta_{st}}{(\Psi_t + \Sigma_{st})^2} \right] = 0, \, \beta_{st} =\left( \vW E[\vz_s \vz_s^T] \vW^T \right)_{tt} +2 (m_t - m_{st}) \left( \vW E[\vz_s] \right)_t + (m_t - m_{st})^2
+\alpha_t &= \frac{\partial \mathcal{L}}{\partial \Psi_t} = -\frac{1}{2}\sum_s M_{st}\left[\frac{1}{\Psi_t + \Sigma_{st}} - \frac{B_{st}}{(\Psi_t + \Sigma_{st})^2}\right],\nonumber\\
+\beta_t &= \frac{\partial^2 \mathcal{L}}{\partial \Psi_t^2} = +\frac{1}{2}\sum_s M_{st}\left[\frac{1}{(\Psi_t + \Sigma_{st})^2} - \frac{2B_{st}}{(\Psi_t + \Sigma_{st})^3}\right].
 \end{align}
-Alternatively, the aforementioned approximation $\vSigma_s \rightarrow \bar{\vSigma}$ gives
-%
+The Newton's approximate solution is therefore:
 \begin{equation}
-\Psi_{tt} = \frac{1}{S} \sum_s \beta_{st} - \bar{\Sigma}_{tt}, \, {\rm (simplified)}
+\Psi_{t,1} = \Psi_{t,0} - \frac{\alpha_t(\vPsi_0)}{\beta_t(\vPsi_0)}.
 \end{equation}
-%
+One may start iterations using the result of Scheme 1 as the initial guess and continue until convergence.\\
 
-In the M step equation for $\vW$ we incorporate an automatic relevance determination (ARD) prior:
+\noindent{\bf Remark:} In practice, when sample-to-sample variables of $\Sigma$ was large (e.g. read depths varying from 50 to 1000 randomly), we noticed that the best approach was to use Brent root finding for each target. On average, 10 function calls yields to solution within a $10^{-6}$ tolerance. The Newton's method required approximately 20 evaluations of $\alpha_t$ and $\beta_t$ to convergance within the same tolerance. Also, we found that the most robust scheme was to start from $\Psi_t = 0$ rather than using Eq.~\eqref{eq:psi_approx_2}.\\
+
+
+In the M step equation for $\vW$, we may incorporate an automatic relevance determination (ARD) prior:
 %
 \begin{align}
-P(\vW) =& \prod_k \left( \frac{\alpha_\mu}{2 \pi} \right)^{T/2} \exp \left( -\frac{1}{2} \alpha_\mu \sum_t W_{t\mu}^2  \right).
+P(\vW) =& \prod_\mu \left( \frac{\alpha_\mu}{2 \pi} \right)^{T/2} \exp \left( -\frac{1}{2} \alpha_\mu \sum_t W_{t\mu}^2  \right).
 \end{align}
 %
-If $\alpha_\mu \rightarrow \infty$ latent feature $\mu$ is effectively turned off.  Thus we can initially choose a liberal estimate of $D$ and the model will automatically become more parsimonious.  The M step log likelihood times the ARD prior depend on the $t$-th row $\vW_{t \cdot}$ of $\vW$ as
+If $\alpha_\mu \rightarrow \infty$, latent feature $\mu$ is effectively turned off. Thus we can initially choose a liberal estimate of $D$ and the model will automatically become more parsimonious. The M step log likelihood times the ARD prior depend on the $t$-th row $\vW_{t \cdot}$ of $\vW$ as:
```

Comment:
I agree -- I got rid of $\vW_{t \cdot}$ in favor of explicitly indexed matrices.